docs/content/docs/start/quick-start.md

```diff
@@ -66,24 +62,42 @@ action copy the full URL in the browser, including the ID, into a new window or
 second client for your dice roller application. With both windows open, click the **Roll** button in either and note
 that the state of the dice changes in both clients.
 
-🥳**Congratulations**🎉 You have successfully taken the first step towards unlocking the world of Fluid collaboration.
+## Running against the Azure Fluid Relay service
+
+To run against the Azure Fluid Relay service, you'll make a code change to ```app.ts```. The app is configured to use a
+local in-memory service called Tinylicious, which runs on port 7070 by default.
+
+To use an Azure Fluid Relay instance instead, replace the configuration values with your Azure Fluid Relay tenant ID,
+orderer, and storage URLs that were provided as part of the FRS onboarding process. Then pass that configuration object
```

Comment:
Would we also want to add the key in here as part of the instructions?

Reply:
I'm hoping that the TokenProvider section will help explain what's happening. I really want to make sure ITP doesn't make it into prod. :)